cypress/support/commands.js

```diff
@@ -33,3 +35,48 @@ Cypress.Commands.add("openRegisterDialogue", () => {
     cy.get('.dropdown-toggle').contains('LOGIN').click();
     cy.get('a.btn-block').contains('Create new account').click();
 })
+
+Cypress.Commands.add("openUploadTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Upload").click();
+    });
+})
+
+Cypress.Commands.add("openListsTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Lists").click();
+    });
+})
+
+Cypress.Commands.add("openTemplatesTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Templates").click();
+    });
+})
+
+Cypress.Commands.add("openRegionsTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Regions").click();
+    });
+})
+
+Cypress.Commands.add("openQueryBuilderTab", () => {
+    cy.get("#bluegenes-main-nav").within(() => {
+        cy.contains("Query Builder").click();
+    });
+})
+
+Cypress.Commands.add("searchKeyword", (keyword) => {
+    cy.get(".search").first().type(keyword + '{enter}',{delay:100});
+})
+
+Cypress.Commands.add("createList", () => {
```

Comment:
Does this work? I can't see it used anywhere so I'm not sure. Those commands inside `cy.get("#bluegenes-main-nav").within` seem to apply to elsewhere than the navbar. It would be better to use `cy.get(".wizard").within`. The selector for adding the identifiers could possibly be simplified to `cy.get('textarea')`.